### PR TITLE
[main] Update tcl and psmisc

### PR DESCRIPTION
**Pull request by christopherco**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update tcl and psmisc packages to their latest released versions. Also verified license for openscap.

###### Change Log  

- Update tcl and psmisc SPECS to latest versions
- Update cgmanifest to reflect the new versions
- Verify license for openscap

###### Does this affect the toolchain?  

**YES**

###### Test Methodology

local build

### CODE REVIEW

**Comment by christopherco** [comment]
Will rebase changes once #1643 is merged

File: SPECS/tcl/tcl.spec [hunk 1]
```diff
@@ -1,16 +1,14 @@
+%define majorver 8.6
```
Comment (PawelWMS) [inline SPECS/tcl/tcl.spec:1]: Nit: might be worth replacing with macros, which automatically extract this value. Example from `kf5-kconfig.spec`:

```suggestion
%define majmin %(echo %{version} | cut -d. -f1-2)
```

Comment (christopherco) [reply SPECS/tcl/tcl.spec]: Great suggestion. Fixed in newest version.

File: SPECS/tcl/tcl.spec [hunk 2]
```diff
@@ -1,16 +1,14 @@
+%define majorver %(echo %{version} | cut -d. -f1-2)
```
Comment (christopherco) [inline SPECS/tcl/tcl.spec:1]: @thcrain-msft Linter is looking to put this macro below between `Source0:` and `BuildRequires:`. Seems to be an odd place to put the macro. What are your thoughts?

Comment (thcrain-msft) [reply SPECS/tcl/tcl.spec:1]: Very odd, haven't seen this behavior before. Feel free to ignore the linter on that. I have some time set aside today to look at linter issues, and I'll add this to my list.

**Review by thcrain-msft — APPROVED** [review]

**Pull request merged**